app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
```

Comment:
If you will go with devise token auth, then it would be best to  drop this model and regenerate it with  devise token auth gem, so it is configured properly. Same with migration for it.

Reply:
@rafalrothenberger for now I decide not to use devise token auth, but only 'devise '

Reply:
If you are going with just `devise` then your authentication is based on cookies, which means you will need to protect your API against [CSRF attacks](https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html).